server/start-dev.js

```diff
@@ -30,16 +40,13 @@ nodemon
   })
   .on('restart', function (files) {
     console.log('App restarted due to: ', files);
-  });
-
-// Build the client-side JavaScript bundle when we see any changes
-build(
-  mergeOptions(viteConfig, {
-    build: {
-      // Rebuild when we see changes
-      watch: true,
-    },
   })
-);
-
-console.log('start-dev2 process.env.NODE_ENV', process.env.NODE_ENV);
+  .on('crash', function () {
+    console.log('Nodemon: script crashed for some reason');
+  })
+  // .on('watching', (file) => {
+  //   console.log('watching');
+  // })
+  .on('log', function (data) {
+    console.log(`Nodemon logs: ${data.type}: ${data.message}`);
+  });
```

Comment:
This will give us all of the `--verbose` logging you would normally see using the CLI